### PR TITLE
add private images doc for compose

### PR DESCRIPTION
**Pull request by taylordolan**

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -3,6 +3,15 @@
 ---
 <!-- {% include breadcrumbs.html %} -->
 <article class="article">
+  <!-- {% if page.development %} -->
+  <div class="grid-block well gray well-in-development">
+    <img class="grid-block shrink icon" src="{{ site.baseurl }}/images/in-development.svg" width="45" height="45">
+    <div class="grid-block vertical">
+      <div class="weight-strong">In Development</div>
+      <div class="weight-light">We’re hard at work on this feature. <br class="hidden-sm">You can expect it soon!</div>
```
Comment (runnabro) [inline _layouts/page.html]: I think I prefer this without the last line.

screen shot 2017-05-17 at 11 06 17 am

File: docker-compose/_posts/2017-05-12-using-private-docker-images.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+```yaml
+Services:
+  db:
+    image: example-registry.com:4000/postgresql
+```
+
+> **Tip:** Learn how to [use your Compose file]({{ site.baseurl }}/docker-compose/launching-your-first-environment) in Runnable.
+
+You can also use images in your services’ Dockerfiles:
```
Comment (runnabro) [inline docker-compose/_posts/2017-05-12-using-private-docker-images.md]: I think it's bit ambiguous if this is intentional. Did you intend to refer to multiple services, or just a single one? I would just drop the word.

File: docker-compose/_posts/2017-05-12-using-private-docker-images.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Open the Private Registry settings for your team. Click on your team's avatar in the upper-left corner and select **Private Registry**.
+
+    ![](images/ss-registry-navigation.png)
+
+2. Enter the credentials for your registry and click **Verify & Save**.
+
+    ![](images/ss-registry-credentials.png)
+
+Now images from your registry that are referred to in your Compose files or Dockerfiles can be used successfully.
```
Comment (runnabro) [inline docker-compose/_posts/2017-05-12-using-private-docker-images.md:43]: Should we show an image for the final state?

File: docker-compose/_posts/2017-05-12-using-private-docker-images.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+development: true
+---
+
+Many teams use private Docker registries to host their images. To use your private images, refer to them in your Compose files or Dockerfiles and add your registry credentials to Runnable.
+
+---
+
+## Using Images in Your Files
+
+Use images in a Compose file by referring to them with the `image` configuration option:
```
Comment (runnabro) [inline docker-compose/_posts/2017-05-12-using-private-docker-images.md]: I would call it the "…`image` feature:`, as that's what the Compose docs calls them.

File: _sass/components/wells.scss [hunk 5]
```diff
@@ -91,3 +91,34 @@
...
+  padding: $xs $sm;
+
+  @include media(sm) {
+    width: 100%;
+  }
+
+  @include media(xxs) {
+    align-items: flex-start;
+    flex-direction: column;
+  }
```
Comment (runnabro) [inline _sass/components/wells.scss]: @iPhone SE size. Should these items be center aligned at very small sizes? Should we use a negative margin hack here to expand the width?

screen shot 2017-05-17 at 11 11 03 am

Comment (taylordolan) [reply _sass/components/wells.scss]: Updated with center-alignment. Decided not to do the negative margin thing though. I think this element looks okay as-is, but at some point I think we could revisit our overall responsive margins at this size.

**Review by runnabro — APPROVED** [review]

**Pull request merged**